config.go

```diff
@@ -18,6 +19,7 @@ import (
 const inMemory = "memory"
 const postgreSQL = "postgres"
 const dynamoDB = "dynamo"
+const SQLite3 = "sqlite3"
```

Comment:
This should probably follow the pattern of the rest of the constants: lowerCamelCase.

Reply:
I actually remember being confused about this, hah. I didn't realize that the uppercase first letter made the const exported. I just went with `sqLite3` in 23a1f23. A little weird, but gets the job done (: